servicemap/schema.py

```diff
@@ -40,16 +38,35 @@ def normalize_service_unit_node(units: list):
     @staticmethod
     def resolve_schools_and_kindergartens_list(parent, info, **kwargs):
         response = api_client.list_helsinki_schools_and_kindergartens(
+            # Use the max page size when the pages are queried
+            # to make minimun amount of API requests.
             filter_list=kwargs
+            or {"page_size": 1000}
         )
         json_data = json2obj(response.content)
+
         normalized_data = Query.normalize_service_unit_node(json_data.results)
+        next_page_url = json_data.next
+
+        if json_data.next:
+            # Recursive call to fetch a next page data from a paginated result list.
+            next_page_response = Query.resolve_schools_and_kindergartens_list(
+                parent, info, **get_params_from_url(json_data.next)
+            )
+            normalized_data.extend(next_page_response.data)
+
+            # Because all pages are combined,
+            # the next_page_url should be set to None
+            # after the last response is retrieved.
+            next_page_url = None
+
         result = {
             "meta": {
                 "count": json_data.count,
-                "next": json_data.next,
+                "next": next_page_url,
```

Comment:
If my brain is working correctly this morning, I think you should be able to just do `"next": None` here. :thinking: